MalwareAnalyzer/views/domain_check.py

```diff
@@ -83,7 +83,9 @@ def malware_check(urllist):
                     details_dict["desc"] = enlist[4]
                     details_dict["bad"] = "yes"
                     for domain in domainlist:
-                        if (details_dict["domain_or_url"].startswith(domain) or
+                        if ((details_dict["domain_or_url"].startswith(domain) or
+                                (domain.startswith(details_dict["domain_or_url"], 4) and
+                                 len(details_dict["domain_or_url"]) > 1)) or
```

Comment:
I think a better code will be 

```if domain in details_dict["domain_or_url"] or details_dict["domain_or_url"] in domain```

Reply:
will test :) need to ensure we will not match domain containing a - cause the malware domain list  contain a - for domain when there's only an IP as info

Reply:
like this it works : 
`if domain in details_dict["domain_or_url"] or 
    (details_dict["domain_or_url"] in domain and (len(details_dict["domain_or_url"]) > 1)) or 
    details_dict["ip"].startswith(domain):`